python/graphstorm/model/utils.py

```diff
@@ -402,11 +423,12 @@ def distribute_nid_map(embeddings, rank, world_size,
     if isinstance(embeddings, (dgl.distributed.DistTensor, LazyDistTensor)):
         # only host 0 will load node id mapping from disk
         if rank == 0:
-            if node_id_mapping_file.endswith("dgl"):
-                # node id mapping file from dgl tools/distpartitioning/convert_partition.py.
-                ori_node_id_mapping = dgl.data.utils.load_tensors(node_id_mapping_file)
-            else: # endswith "pt"
+            if node_id_mapping_file.endswith("pt"):
                 ori_node_id_mapping = th.load(node_id_mapping_file)
+            else:
+                # Homogeneous graph
+                # node id mapping file from dgl tools/distpartitioning/convert_partition.py.
+                ori_node_id_mapping = _load_dist_nid_map(node_id_mapping_file, ["_N"])["_N"]
```

Comment:
Are we sure the node types are always going to be that?

Reply:
For distDGL, yes.